docs/react-testing-library/migrate-from-enzyme.mdx

```diff
@@ -330,10 +330,14 @@ input's "checked" property is properly set. Let's see how we might write a test
 for that case:
 
 ```jsx
-test('handles click correctly', () => {
+test('handles click correctly', async () => {
   render(<Checkbox />)
+  const user = userEvent.setup()
+
+  // You can also call this method directly on userEvent,
```

Comment:
I'm not sure if that's even something we want to "share". If we believe that the recommended approach is to use the methods returned from `setup`, maybe this comment isn't helpful and can only cause bad practices. Wdyt?

Reply:
I think it eases the transition for people who are used to the previous APIs. That's why the [Direct APIs](https://testing-library.com/docs/user-event/setup#direct-apis) still exist.

If there is only a single API call, it makes no difference, as the Direct APIs call `.setup()` internally and then use the method from there.

If it helps or hurts to mention it in the example - I don't know.